docs/OntologiesReviewWorkflow

```diff
@@ -0,0 +1,42 @@
+---
+layout: doc
+title: Ontology Review Process
+---
+
+This page deals with the process, policies, and guidelines for manually reviewing newly-submitted ontologies, with particular focus on operational aspects such as:
+
+- which ontologies are reviewed, and in what order?
+- how are reviewers chosen?
+- what is the workflow for conducting a review?
+- what are the criteria used?
+
+Ontologies are reviewed manually after completing the automated evaluation. Though the automated evaluation checks for adherence to [OBO Foundry principles](http://www.obofoundry.org/principles/fp-000-summary.html), it cannot capture certain nuances outlined in the principles nor evaluate ontology quality. The purpose of the manual review is to check the ontology for issues that the NOR Dashboard does not cover.
```

Comment:
>  Ontologies are reviewed manually after completing the automated evaluation.

I guess "completing" implies "passing" but would it be good to state that explicitly? If the submitted ontology completes the automated evaluation and fails anything, it is not reviewed manually yet, right?

Reply:
Might also be good to clarify by saying "Newly-submitted ontologies" (even though it does say that in the first sentence).

Reply:
> The purpose of the manual review is to check the ontology for issues that the NOR Dashboard does not cover.

This document doesn't seem to cover what happens after an ontology passes (or fails) the review. I assume that is intentional, but it affects things like my comment below that a passing ontology needs to say yes to (as currently worded) some of the criteria.

Reply:
Yes, completing was intended to mean passing. Fixed.
Also clarified that this applies to new ontologies as suggested.
I did indeed intentionally leave out what happens upon passing or failing, as this is covered elsewhere and is technically not part of the review process. This doc is intended to stay focused on the review process itself.